assets/javascripts/kitten/components/box/doc-link-box.js

```diff
@@ -4,7 +4,7 @@ import LinkBox from './link-box'
 export default class DocLinkBox extends React.Component {
   render() {
     return (
-      <LinkBox { ...this.props } displayIcon={ true } >
+      <LinkBox { ...this.props } displayIcon={ true }>
```

Comment:
Je crois que la version `<LinkBox { ...this.props } displayIcon>` marche aussi si la prop est toujours à `true`.